app/web/src/newhotness/ActionCard.vue

```diff
@@ -233,19 +233,25 @@ const hold = () => {
 const holdApi = useApi();
 const finishHold = async () => {
   const call = holdApi.endpoint(routes.ActionHold, { id: props.action.id });
+
+  // This route can mutate head, so we do not need to handle new change set semantics.
   await call.put({});
```

Comment:
You'll see this in all locations where `put`, `post`, and `delete` calls are used for `CAN_MUTATE_ON_HEAD` routes. This should not be the default pattern for developers. These routes are the exception and we should notate it when working with them.